src/SourceBuild/content/build.proj

```diff
@@ -34,6 +34,27 @@
     <RemoveDir Directories="$(BaseOutputPath)" />
   </Target>
 
+  <!-- Copies the output assets of the builds to the output path. -->
+  <Target Name="CopyBinariesToBinFolder"
+          AfterTargets="Build"
+          Inputs="$(MSBuildProjectFullPath)"
+          Outputs="$(CompletedSemaphorePath)CopyBinariesToBinFolder.complete">
+    <ItemGroup>
+      <_builtSymbolPackages Include="$(SourceBuiltAssetsDir)*.symbols.nupkg" />
+      <_builtSymbolPackages>
+        <TransformedFileName>$([System.String]::Copy('%(FileName)').Replace('symbols', 'nupkg'))</TransformedFileName>
```

Comment:
I'm confused by the replace.  Isn't that going to produce a `<package>.nupkg.nupkg` file name?

Reply:
Can you share the output from a build with symbols?

Reply:
Hmm, yes it would. This is from the original runtime code: https://github.com/dotnet/installer/pull/17164/files#diff-3de696aba794a0ce36c0c339dd1fbc20d57d494f6b161eb005cb1655d71b3079L86-L89

Source-build produces no symbol packages right now, so this code hasn't ever been functional even in its old form. Perhaps just best to remove it for now since it's not correct.

/cc @NikolaMilosavljevic

Reply:
@mmitche do the `runtime` and other repos still produce various `*.symbols.nupkg` in their artifacts folder? Those packages will be used for future work to enable access to PDBs and allow sourcelink validation.

Reply:
> @mmitche do the `runtime` and other repos still produce various `*.symbols.nupkg` in their artifacts folder? Those packages will be used for future work to enable access to PDBs and allow sourcelink validation.

They don't appear to do so in source-build. So that will need figuring out before that code would be active.

Reply:
These packages are produced in full source-build and your changes are not affecting that. Here are few examples of those packages:
```
./src/runtime/artifacts/source-build/self/src/artifacts/packages/Release/Shipping/System.Reflection.Metadata.8.0.0-rc.1.23381.3.symbols.nupkg
./src/sdk/artifacts/source-build/self/src/artifacts/packages/Release/NonShipping/Microsoft.DotNet.Cli.8.0.100-rc.1.23401.23.symbols.nupkg
./src/aspnetcore/artifacts/source-build/self/src/artifacts/packages/Release/Shipping/Microsoft.Extensions.ObjectPool.8.0.0-rc.1.23401.1.symbols.nupkg
```

> > @mmitche do the `runtime` and other repos still produce various `*.symbols.nupkg` in their artifacts folder? Those packages will be used for future work to enable access to PDBs and allow sourcelink validation.
> 
> They don't appear to do so in source-build. So that will need figuring out before that code would be active.

They do produce these packages and I have verified in a local VMR build with your changes, that the functionality is still preserved. Here are just few examples of those packages:
```
./src/runtime/artifacts/source-build/self/src/artifacts/packages/Release/Shipping/Microsoft.Extensions.Logging.Console.8.0.0-rc.1.23381.3.symbols.nupkg
./src/sdk/artifacts/source-build/self/src/artifacts/packages/Release/NonShipping/Microsoft.DotNet.Cli.8.0.100-rc.1.23401.23.symbols.nupkg
./src/aspnetcore/artifacts/source-build/self/src/artifacts/packages/Release/Shipping/Microsoft.Extensions.ObjectPool.8.0.0-rc.1.23401.1.symbols.nupkg
```

`aspnetcore` produces the runtime nupkg with binaries and PDBs alongside:
```
Microsoft.AspNetCore.App.Runtime.fedora.36-x64.8.0.0-preview.7.23355.19.symbols.nupkg
```

Runtime produces a tarball with all PDBs:
```
./artifacts/x64/Release/dotnet-runtime-symbols-fedora.36-x64-8.0.0-rc.1.23381.3.tar.gz
```

I'm going to explore what's needed to enable symbols.nupkg for all `runtime` components, as we want a unified experience for both sourcelink validation and partner PDB consumption.